### PR TITLE
feature: implement i18n

### PR DESCRIPTION
**Pull request by binh1298**
image
image

### CODE REVIEW

File: src/i18n/LanguageSwitcher.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { MenuItem, Select } from "@material-ui/core";
```
Comment (kien123456k) [inline src/i18n/LanguageSwitcher.tsx:1]: Should we create our base component for those 2 components?

File: src/pages/Home/index.tsx [hunk 2]
```diff
@@ -20,15 +24,24 @@ export const HomeUserGraphQL = graphql`
 
 const Home = (props: PagePropsWithQuery<HomeUserQuery>) => {
     const data = usePreloadedQuery<HomeUserQuery>(HomeUserGraphQL, props.queryReference);
+
+    const { t } = useTranslation(["common", "home"]);
```
Comment (kien123456k) [inline src/pages/Home/index.tsx:28]: I think we should use alias to give this variable a more meaningful name.

File: src/pages/Login/index.tsx [hunk 3]
```diff
@@ -91,6 +92,7 @@ const Login = () => {
         window.location.href = redirectUrl;
     };
 
+    const { t } = useTranslation(["login"]);
```
Comment (kien123456k) [inline src/pages/Login/index.tsx:95]: Same here. I think a more meaningful name will be better for this variable.

File: src/pages/Home/index.tsx (hunk 2 above)
Comment (binh1298) [reply src/pages/Home/index.tsx:28]: I'm not sure I understand what you mean, this is typesafe and recommended by i18nnext

Comment (kien123456k) [reply src/pages/Home/index.tsx:28]: oh okay, I didn't know that.

File: src/i18n/LanguageSwitcher.tsx (hunk 1 above)
Comment (kien123456k) [reply src/i18n/LanguageSwitcher.tsx:1]: Okay I will 😅

**Review by binhtt13122000 — APPROVED** [review]
OK with love

**Pull request merged**